ts/Extensions/Annotations/Annotations.ts

```diff
@@ -1631,8 +1631,8 @@ H.extendAnnotation = function <T extends typeof Annotation> (
     merge(
         true,
         Constructor.prototype,
-        BaseConstructor.prototype,
-        prototype
+        BaseConstructor.prototype as any,
+        prototype as any
     );
```

Comment:
This should use `extend` not `merge` imho, because merge works recursive.
```ts
    extend(
        Constructor.prototype,
        merge(
            BaseConstructor.prototype,
            prototype
        )
    );
```